apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -75,8 +86,50 @@ const ERRORS = {
         message: '"qualityControlValue" must be specified if there is "qualityControlAdditionalOptions" or "qualityControlComment"',
         messageForUser: 'api.ticket.QUALITY_CONTROL_VALUE_MUST_BE_SPECIFIED',
     },
+    TICKET_FOR_PHONE_DAY_LIMIT_REACHED: {
+        code: 'BAD_USER_INPUT',
+        type: 'TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
+        message: 'TICKET_FOR_PHONE_DAY_LIMIT_REACHED',
```

Comment:
Why message is the same as type? And `messageForUser` must be here, since it resident-oriented error.

Reply:
Yep, messageForUser will be added later, once the texts are ready. It is stated in descr